rtf

```diff
@@ -542,10 +542,10 @@ class Platform:
           curl_cmd = 'curl --insecure --silent --header \\"X-API-Token: \\$(cat /var/rudder/run/api-token)\\"'
           status_cmd = curl_cmd + ' https://localhost/rudder//api/latest/nodes/' + uuid + " | grep -q success"
           accept_cmd = curl_cmd + ' --request POST https://localhost/rudder//api/latest/nodes/pending/' + uuid + ' --data \\"status=accepted\\"'
+          server.run(accept_cmd, quiet=False, live_output=True, fail_exit=fail_exit)
```

Comment:
you want to accept a node before we check that its inventory has been received ?